src/components/editMeasure/details/MeasureDetails.tsx

```diff
@@ -376,7 +372,7 @@ export default function MeasureDetails(props: MeasureDetailsProps) {
               />
             </>
           )}
-          {!isQDM && featureFlags?.QICoreMeasureDefinitions && (
+          {!isQDM && (
```

Comment:
this file still has `QICoreMeasureReferences` flag usage

Reply:
Good catch! I think I got the rest of them